### ISSUE TITLE
ResXFileCodeGenerator does not emit GetTypeInfo()

### ISSUE THREAD

**Issue by fubar-coder** [dotnet/project-system#1519]
# How to reproduce
I have a project that targets both net451 and netstandard1.3 with:

```
    <TargetFrameworks>net451;netstandard1.3</TargetFrameworks>
```

# Expected

The code should be compilable.

# What happened instead

The ResXFileCodeGenerator doesn't emit GetTypeInfo().

# Workaround

I have to move the target framework `netstandard1.3` to the first position to get the expected behavior:
```
    <TargetFrameworks>netstandard1.3;net451</TargetFrameworks>
```

# Additional info

This seems to be related to #747.

**Comment by davkean** [comment]
No, this is different to #747 and a limitation in how the RESX designer queries the project. it's not aware of multiple frameworks - so it gets what we consider the "active" one which is the first one in the list. We'll need to think about how we solve this particular issue (in some cases - it's impossible to generate code that works everywhere).

As a workround, put netstandard first, reload the project and modify the designer to get it to regen the code.

**Comment by AArnott** [comment]
I just hit this and discovered the workaround by accident as well. The workaround is fine. But its discoverability is poor. Let's at least fix that.

**Comment by cyrilcathala** [comment]
Please note the workaround doesn't work if netstandard is the only target:
`<TargetFramework>netstandard1.4</TargetFramework>`

Tested on Visual Studio for Mac 7.0.1.

**Comment by execjosh** [comment]
> Please note the workaround doesn't work if netstandard is the only target:
> <TargetFramework>netstandard1.4</TargetFramework>
> 
> Tested on Visual Studio for Mac 7.0.1.

We are experiencing the same thing on the same Visual Studio for Mac Professional 7.0.1 (build 24).

Our project has only one target framework, and uses `project.json`.

```json
...
  "frameworks": {
    "netstandard1.4": {}
  }
...
```

**Comment by jorgeyanesdiez** [comment]
I can confirm this still happens with Visual Studio 2017.3. and .NET Core SDK 1.0.4

**Comment by shibaev** [comment]
Please note that workaround does not help if one of the target platforms does not support Type.GetTypeInfo(). For example, <TargetFrameworks>netstandard1.3;net40</TargetFrameworks>

**Comment by dmonroym** [comment]
@ocallesp There might be no-action here but it's worth taking a look if there's a fix. The workaround in this issue seems to be doing its job for now
